examples/vacanza.py

```
@@ -0,0 +1,153 @@
+        holidays.country_holidays(country_code)
+    except NotImplementedError:
+        print(f"Error: Country code '{country_code}' is not supported.")
+        print("Visit https://holidays.readthedocs.io for a full list of supported countries.")
+        sys.exit(1)
+
+    # Determine which categories to generate
+    all_categories = get_supported_categories(country_code)
+    if public_only:
+        categories = (PUBLIC,)
+    else:
+        categories = all_categories
```

Ruff (0.15.6)

[warning] Use ternary operator `categories = (PUBLIC,) if public_only else all_categories` instead of `if`-`else`-block

Replace `if`-`else`-block with `categories = (PUBLIC,) if public_only else all_categories`

(SIM108)